stdlib/_bisect.pyi

```diff
@@ -1,51 +1,75 @@
 import sys
-from _typeshed import SupportsRichComparison, SupportsRichComparisonT
+from _typeshed import SupportsRichComparison
 from typing import Callable, MutableSequence, Sequence, TypeVar, overload
 
-_T = TypeVar("_T")
+_SupportsRichComparisonT = TypeVar("_SupportsRichComparisonT", bound=SupportsRichComparison)
 
 if sys.version_info >= (3, 10):
+    _KeyReturnT = TypeVar("_KeyReturnT", bound=SupportsRichComparison)
+
     @overload
-    def bisect_left(a: Sequence[_T], x: _T, lo: int = ..., hi: int | None = ..., *, key: None = ...) -> int: ...
+    def bisect_left(
+        a: Sequence[_SupportsRichComparisonT],
+        x: _SupportsRichComparisonT,
+        lo: int = ...,
+        hi: int | None = ...,
+        *,
+        key: None = ...,
+    ) -> int: ...
     @overload
     def bisect_left(
-        a: Sequence[_T],
-        x: SupportsRichComparisonT,
+        a: Sequence[_SupportsRichComparisonT],
```

Comment:
Why do we still need a bounded TypeVar in this overload? If there's a key function, we don't need to support comparisons on the list elements themselves, right?